e2e-tests/ukef/cypress/support/portal/makerSubmitDealForReview.js

```diff
@@ -0,0 +1,13 @@
+import portalPages from '../../../../portal/cypress/e2e/pages';
+import MOCK_USERS from '../../../../e2e-fixtures/portal-users.fixture';
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+export const makerSubmitDealForReview = (deal) => {
+  cy.login(BANK1_MAKER1);
+  portalPages.contract.visit(deal);
+  portalPages.contract.proceedToReview().click();
+
+  cy.keyboardInput(portalPages.contractReadyForReview.comments(), 'go');
+  portalPages.contractReadyForReview.readyForCheckersApproval().click();
+};
```

Comment:
Nice. This and `checkerSubmitDealToUkef` will be great and help in this [draft/WIP PR](https://github.com/UK-Export-Finance/dtfs2/pull/3594/files#diff-caed56733578614ed227f6f8d0f10bb30594077a1fb93741b68f139782c7e9dbR38) @vamshidharukef 

Can we add some documentation please

Reply:
@ttbarnes Do you mean comments what kind of documentation do you mean ?

Reply:
E.g

```js
/**
 * Login as a maker and submit a deal for review
 * @param ...
 */
```

Some fuller exmples here 😊  https://github.com/UK-Export-Finance/dtfs2/tree/main/e2e-tests/tfm/cypress/support/click-events